src/1-raw-data/mailinglists/main.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+This package is for loading different mailing list data types into Cloud Storage.
+*/
+
+package main
+
+//TODO
+// Check the most recent file stored and pull only what isn't there
+
+import (
+	"cloud.google.com/go/storage"
+	"context"
+	"flag"
+	"fmt"
+	"google.golang.org/api/iterator"
+	"io"
+	"log"
+	"net/http"
+)
+
+type storageConnection struct {
+	ctx    context.Context
```

Comment:
Consider having the context be the first argument to the functions that use it (i think this is idiomatic), it gives the caller greater flexibility